index.html

```diff
@@ -0,0 +1,533 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet"> 
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <main>
+        <nav class="header addBorder">
+            <div class="pageLogo headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Logo</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Item 1</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Item 2</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Item 3</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Item 4</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Item 5</h2></a> </div>
+            <div class="headerItem addBorder"> <a href="#" class="whiteTextNoStyle"><h2>Search</h2></a> </div>
+        </nav>
```

Comment:
- [ ] The structure of your project should have three main parent tags: nav/header, main and footer. Please take the nav tag outside the main tag.